packages/fxa-auth-server/scripts/verification-reminders.js

```diff
@@ -29,6 +29,8 @@ const verificationReminders = require(`${LIB_DIR}/verification-reminders`)(
   config
 );
 
+const cadReminders = require(`${LIB_DIR}/cad-reminders`)(config, log);
```

Comment:
Maybe this file should be renamed to `reminders.js` or `email-reminders.js` now that it covers CAD and verification reminders?

Reply:
yes, if this experiment succeeds we have several tasks to refactor and improve the architecture. the goal at this point is to do the least possible so that it's easy to revert.